src/main/python/covata/delta/apiclient.py

```diff
@@ -229,7 +225,7 @@ def get_secret(self, requestor_id, secret_id):
         :param str requestor_id: the authenticating identity id
         :param str secret_id: the secret id to be retrieved
         :return: the retrieved secret
-        :rtype: dict[str, any]
+        :rtype: dict[str, str]
```

Comment:
Oh - good pickup, yes I'll review those ones, pretty sure you are correct.

It doesn't matter if they remain dict[str,any] anyway even if it was dict[str, str]